### ISSUE TITLE
📎 Create GitHub actions to publish rust crates

### ISSUE THREAD

**Issue by magic-akari** [biomejs/biome#286]
### Description

- https://github.com/biomejs/biome/discussions/190

**Comment by nissy-dev** [comment]
I started working. I'm learning how to publish multiple crates by cargo-workspace and manage crate.io token in my fork repository.

- https://github.com/pksunkara/cargo-workspaces
- https://doc.rust-lang.org/cargo/reference/publishing.html#cargo-owner

After that, I will work in the following way (this may change)

- Try to publish on your own account
- Change crate's owner to github's team if publishing well
- Set up a GitHub Actions workflow

**Comment by ematipico** [comment]
Few things I want to share:
- we don't want to maintain a `CHANGELOG.md` for each crate. While it seems nice, it's a a lot of burden, and we still need to get current changelog right
- `cargo-workspace` is a nice tool, but when I tried it, it didn't work as expected. I wanted to do a "dry run" locally, so I disabled everything (github tags, commit, push, etc.), and regardless of that, it **published** the crate on `crates.io`;
- it's fine we start with a simple script that does the publishing, e.g. https://github.com/tokio-rs/tokio/blob/master/bin/publish
